src/Tizen.NUI.Components/Controls/Button.cs

```diff
@@ -1088,15 +1107,15 @@ protected virtual void MeasureText()
             }
             buttonText.WidthResizePolicy = ResizePolicyType.Fixed;
             buttonText.HeightResizePolicy = ResizePolicyType.Fixed;
-            int textPaddingLeft = buttonAttributes.TextAttributes.PaddingLeft;
-            int textPaddingRight = buttonAttributes.TextAttributes.PaddingRight;
-            int textPaddingTop = buttonAttributes.TextAttributes.PaddingTop;
-            int textPaddingBottom = buttonAttributes.TextAttributes.PaddingBottom;
+            int textPaddingLeft = buttonAttributes.TextAttributes.Padding.Start;
```

Comment:
xxxLeft and xxxRight are not good for RTL language.   Please use xxxStart and xxxEnd instead.

Reply:
I've modified it.